cmd/root.go

```diff
@@ -130,20 +132,29 @@ func parseConfig(cmd *cobra.Command, conf *proxy.Config, args []string) error {
 	if len(args) == 0 {
 		return newBadCommandError("missing instance_connection_name (e.g., project:region:instance)")
 	}
-	// First, validate global config.
+
+	userSet := func(f string) bool {
```

Comment:
nit: this might be more clear as "userHasSet", on my first take I thought it was a set of user info

Reply:
Sounds good to me.